sdk/identity/identity/review/identity.api.md

```diff
@@ -74,6 +74,10 @@ export class AzureCliCredential implements TokenCredential {
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken>;
 }
 
+// @public
+export interface AzureIdentityExtensionTypeMap {
```

Comment:
this pattern is interesting, the whole "we declare an empty map and typings files from other packages add to it", I'm not sure I've seen it before. Is there a reason we need to do it this way, rather than figuring out some kind of interface contract?

Reply:
It only helps us hide implementation details from the API, but this was removed in favor of simply using `(context: unknown) => void` to represent an extension. Extensions now cast `context` to `AzureExtensionContext`, which they get through an internal import.